README.md

```diff
@@ -95,7 +80,3 @@ You will also need the UID of a testing user to authenticate your tests. Create
 First run `npm run emulators:CI` to run the emulators using the testing data in cypress-emulator-data.
```

Comment:
This line should be removed. Since it's a mobile app now we don't have Cypress tests and the script `npm run emulators:CI` doesn't exist anymore.